lib/src/main/java/com/basistheory/android/view/TextElement.kt

```diff
@@ -112,6 +114,20 @@ open class TextElement @JvmOverloads constructor(
         }
     }
 
+    fun setValueRef(
+        vararg elementValueReferences: ElementValueReference,
+        transform: ((String?) -> CharSequence)? = null
```

Comment:
This is a little unexpected... was this to account for setting expiration date? 

I wonder how this compares to how ios/web are implemented.

Reply:
yes, it enables setting multiple value references and transforming their values to be able to set the expiration date